src/components/widgets/SelectWidget.js

```diff
@@ -3,18 +3,16 @@ import PropTypes from "prop-types";
 
 import { asNumber } from "../../utils";
 
+const nums = new Set(["number", "integer"]);
```

Comment:
Shouldn't this `Set` be taken from `core-js`? Or maybe the same change as above, using the polyfilled `includes` from `core-js`.